CMakeLists.txt

```diff
@@ -6,7 +6,7 @@ cmake_minimum_required(VERSION 3.9)
 ##---------------------------------------------------------------------------##
 # Define your project name and set up major project options
 ##---------------------------------------------------------------------------##
-project(Cabana LANGUAGES CXX)
+project(Cabana LANGUAGES CXX Fortran)
```

Comment:
Move it to Fortran example option, so that we don’t require Fortran by default.

Reply:
Ditto

Reply:
Done

Reply:
Please remove Fortran here.

Reply:
Please remove Fortran here.

Reply:
I thought I did. Now done.